apps/backend/src/authn/authn.service.ts

```diff
@@ -26,6 +28,37 @@ export class AuthnService {
     }
   }
 
+  async oauthValidateUser(
+    email: string,
+    firstName: string,
+    lastName: string
+  ): Promise<User> {
+    let user: User;
+    try {
+      user = await this.usersService.findByEmail(email);
+    } catch {
+      const randomPass = crypto.randomBytes(128).toString('hex');
+      const createUser: CreateUserDto = {
+        email: email,
+        password: randomPass,
+        passwordConfirmation: randomPass,
+        firstName: firstName,
+        lastName: lastName,
+        organization: '',
+        title: '',
+        role: 'user'
```

Comment:
Looks like this is supposed to have a default value on the model, not sure what was causing it to not be set (not saying to change this, just definitely weird behavior and may be worth investigating further since it popped up on the LDAP PR as well)

Reply:
Yeah it definitely should be setting it by default, the only reason I can think of why it doesn't happen is if sequelize handles the AllowNull(false) decorator weirdly.